meta-mender-octavo-osd32mp/conf/machine/include/osd32mp1.inc

```diff
@@ -0,0 +1,45 @@
+include conf/machine/include/st-machine-common-stm32mp.inc
+include conf/machine/include/st-machine-providers-stm32mp.inc
+
+# Define specific familly common machine name
+MACHINEOVERRIDES .= ":stm32mp1common:stm32mp15common"
+
+# =========================================================================
+# Chip architecture
+# =========================================================================
+DEFAULTTUNE = "cortexa7thf-neon-vfpv4"
+include conf/machine/include/arm/armv7a/tune-cortexa7.inc
+
+# =========================================================================
+# boot scheme
+# =========================================================================
+BOOTSCHEME_LABELS += "trusted"
+#BOOTSCHEME_LABELS += "optee"
```

Comment:
very nitpicky - feel free to ignore: maybe add a small comment that explains why "trusted", not "optee", and if optee can be used under which prerequisites.

Reply:
I added a small note. Only trusted for now for the sake of simplicity.
Enabling optee looks like a nice topic for future work, maybe when reuniting osd32mp1 and stm32mp layers.